cmd/discovery/client/config.go

```diff
@@ -28,12 +27,14 @@ type Config struct {
 func unmarshalConfig() (Config, error) {
 	var result Config
 
-	o := &collector.Options{NoLogs: true}
+	o := &collector.Options{NoLogs: true, MergeBootCMDLine: false}
```

Comment:
This isn't changing anything right? We never used cmdline for kcrypt-challenger. We are just being explicit now?

Reply:
Yep. Cmdline should only be applicable to cloud config stuff like the agent. 

In any case, this should default to false on te struct but I wanted to be very explicit as this now requires a filter function and I'm passing an empty one that does nothing, and that is only triggered if the cmdline is parsed